Jenkinsfile

```diff
@@ -67,7 +67,7 @@ podTemplate(
 
         stage('Cleanbuild') {
             withEnv(["MIX_ENV=test", "DATABASE_URL=${DATABASE_URL}"]) {
-                sh("mix do compile --warnings-as-errors --force, test --exclude test")
+                sh("mix do clean, compile --warnings-as-errors --force, test --exclude test")
```

Comment:
[out of curiosity]: I though it fetched code on clear instance and build from scratch, or this `clean` does smth which helps here ❓

Reply:
code is already built in this moment, since `Unit test` built it in `MIX_ENV=test` env. The other order of execution on CircleCI made the issue hit there, but not here